content/patterns/button/examples/button.html

```diff
@@ -195,6 +195,18 @@ <h2 id="sc1_label">HTML Source Code</h2>
           sourceCode.make();
         </script>
       </section>
+
+
+      <section>
+        <h2>Support Levels</h2>
+        <iframe
+          class="support-levels-command-button"
```

Comment:
Curious about the name of this class. I didn't look at the class definition, but assume that it could be used for all AT support iframes so should not include -command-button?

Reply:
This seems to be needed. Based on the usage from the ARIA-AT App at [aria-at-app/server/handlebars/public/script.js L30:L34](https://github.com/w3c/aria-at-app/blob/1885ac1fce9c38a0889801b6d1a6d0f8056805b3/server/handlebars/public/script.js#L30:L34), it posts a message to the APG example webpage for each `iframe` (with their unique class name) that's used to maintain a fixed height which prevents a scrollbar on the `iframe`s. This is seen at [aria-practices/content/shared/js/app.js L51:L60](https://github.com/w3c/aria-practices/blob/575926a48cb14d8497ff3bdba1889d20791b1c6e/content/shared/js/app.js#L51:L60).

It could also be general class name such as `support-levels-pattern` loop through those elements to fix the height but it would mean that when multiple `iframe`s are included, some would be evaluated again as it happens for each `iframe` that's included.

Reply:
Ah, understand now. thank you.